app/services/bulk_upload/lettings/year2024/row_parser.rb

```diff
@@ -0,0 +1,1468 @@
+class BulkUpload::Lettings::Year2024::RowParser
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include InterruptionScreenHelper
+
+  QUESTIONS = {
+    field_1: "Which organisation owns this property?",
```

Comment:
I haven't checked these, happy to trust you if you're happy they're right 😅

Reply:
Yeah, there were a lot of mappings, I hope they're right and I don't expect you to check every single one of them 😅